common/mongoClient.py

```diff
@@ -0,0 +1,153 @@
+from pymongo import MongoClient
+from pymongo.database import Database
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+# HARDCODED collection names
+feedback_collection_name = "feedback"
+context_collection_name = "context"
+
+print(os.getenv("MONGO_DB_URI"))
+
+
+def get_mongo_client() -> Database:
```

Comment:
Returns Database or MongoClient?

Reply:
ah , So when i was first making the design, I wanted it to return the mongoClient, but later on it was rarely used apart from just getting the database client instance, so I changed it, I'll change the semantics and fix this